AFQ/segmentation.py

```diff
@@ -219,6 +222,7 @@ def _seg_afq(self, bundle_dict, streamlines, fdata=None, fbval=None,
         Quantification" PloS One 7 (11): e49790.
         """
         self.logger.info("Preparing Segmentation Parameters...")
+        self.img_affine = img_affine
```

Comment:
Maybe we need to add some error handling: if users provide both `img_affine` and fdata, that could result in conflicting information.  So maybe users should  be allowed to provide one, or the other, but not both?